api/src/main/java/com/epam/esm/controller/GiftController.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.model.CustomSearchRequest;
+import com.epam.esm.model.dto.GiftCertificateDto;
+import com.epam.esm.service.GiftService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
it is better to avoid * in imports as for developers it isn't obvious what exact imports are used in class